tests/test_download.py

```diff
@@ -1,22 +1,41 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
 # Copyright (c) 2021 LG Electronics Inc.
 # SPDX-License-Identifier: Apache-2.0
+
+import os
+
+import pytest
+
 from fosslight_util.download import cli_download_and_extract
+from tests import constants
+
+
+def test_download_from_github():
+    # when
+    target_dir = os.path.join(constants.TEST_RESULT_DIR, "download/example")
+    success, _, _, _ = cli_download_and_extract("https://github.com/LGE-OSS/example",
+                                                target_dir,
+                                                "test_result/download_log/example")
+
+    # then
+    assert success is True
+    assert len(os.listdir(target_dir)) > 0
 
 
-def main():
-    success, msg, _, _ = cli_download_and_extract("https://github.com/LGE-OSS/example",
-                                                  "test_result/download/example",
-                                                  "test_result/download_log/example")
-    if not success:
-        raise Exception(f"Download failed with git:{msg}")
-    success, msg, _, _ = cli_download_and_extract("https://pypi.org/project/filelock/3.4.1",
-                                                  "test_result/download/filelock",
-                                                  "test_result/download_log/filelock")
-    if not success:
-        raise Exception(f"Download failed with wget:{msg}")
+@pytest.mark.parametrize("project_name, project_url",
+                         [("filelock", "https://pypi.org/project/filelock/3.4.1"),
+                          ("dependency", "https://pypi.org/project/fosslight-dependency/3.0.5/"),
+                          ("jackson", "https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.12.2"),
+                          ("pub", "https://pub.dev/packages/file/versions/5.2.1")])
+def test_download_from_wget(project_name, project_url):
+    # given
+    target_dir = os.path.join(constants.TEST_RESULT_DIR,
+                              os.path.join("download", project_name))
+    log_dir = os.path.join(constants.TEST_RESULT_DIR,
+                           os.path.join("download_log" + project_name))
 
+    # when
+    success, _, _, _ = cli_download_and_extract(project_url, target_dir, log_dir)
```

Comment:
상태코드만 사용하실 거면 하기와 같은 방법도 괜찮을 것 같습니다!

```suggestion
    result = cli_download_and_extract(project_url, target_dir, log_dir)
    success = result.success
```

Reply:
오 이렇게도 작성할 수 있군요! 새롭게 배워갑니다ㅎㅎ 
해당 코드 스타일은 기존 레거시 테스트 코드 스타일을 따른 것이기도 하고 개인적으로 메서드 반환 인자가 무엇이 있는지 명시해주는 것이 더 이해하기에는 쉬운 것 같아서 일단은 스타일을 유지하는 쪽으로 해보겠습니다..!